deploy/003-deploy-VBep20.ts

```diff
@@ -15,6 +15,22 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     autoMine: true,
   });
 
+  await deploy("USDT", {
+    contract: "MockToken",
+    from: deployer,
+    args: ["Tether", "USDT", 18],
+    log: true,
+    autoMine: true,
+  });
```

Comment:
Do we need this? We are not deploying a market for USDT in this file?

If we are deploying it for the PSM, we could move it to the PSM deployment script